cfddns.py

```diff
@@ -62,9 +62,12 @@
 
 logger = logging.getLogger()
 logger.setLevel(logging.INFO)
-
+if platform.system().lower() != "linux":
+    platform_log = f"{os.getcwd()}\cfddns.log"
```

Comment:
\ is only for windows, maybe we can do this:

if os.path.exists('/tmp'):
    default_log_path = '/tmp/cfddns.log'
else:
    default_log_path = os.path.join(os.getcwd(), 'cfddns.log')

this code haven't been checked yet, please verify it works